NitroxPatcher/Patches/Dynamic/Player_SetCurrentEscapePod_Patch.cs

```diff
@@ -17,10 +17,11 @@ public static void Prefix(ref Player __instance, EscapePod value)
         {
             NitroxId podId = null;
 
-            if (value != null)
+            if (value && value.GetComponent<NitroxEntity>() != null)
```

Comment:
I think we should not be using null equality checks with components, which is why we have `AliveOrNull`
```suggestion
            if (value && value.GetComponent<NitroxEntity>().AliveOrNull())
```

Reply:
This is fine because Unity overrides the == and != operators. We have an analyzer that will warn you on incorrect usage.